tests/foreman/api/test_repositories.py

```diff
@@ -190,3 +190,29 @@ def test_positive_sync_kickstart_repo(self, module_entitlement_manifest_org, tar
     rh_repo = rh_repo.read()
     assert rh_repo.content_counts['package_group'] > 0
     assert rh_repo.content_counts['rpm'] > 0
+
+
+@pytest.mark.tier1
+def test_positive_multiple_orgs_with_same_repo(target_sat):
+    """Test that multiple organizations with the same repository synced have matching metadata
+
+    :id: 39cff8ea-969d-4b8f-9fb4-33b1ba768ff2
+
+    :Steps:
+        1. Create multiple organizations
+        2. Sync the same repository to each organization
+        3. Assert that each repository from each organization contain the same content counts
+
+    :expectedresults: Each repository in each organziation should have the same content counts
+    """
+    repos = []
+    orgs = [target_sat.api.Organization().create() for _ in range(3)]
+    for org in orgs:
+        product = target_sat.api.Product(organization=org).create()
+        repo = target_sat.api.Repository(
+            content_type='yum', product=product, url=settings.repos.module_stream_1.url
+        ).create()
+        repo.sync()
+        repo_counts = target_sat.api.Repository(id=repo.id).read().content_counts
```

Comment:
We can add more assertions here and just read all the metadata of repos in place of the count.

Reply:
@shweta83 What other metadata would you want to read? my thought is that content_counts is the best assertion for this case.

Reply:
RPM repositories contain a directory with multiple metadata files. This old Pulp doc contains a good summary of each of the files in typical yum repodata under the section `The Yum/RPM Repository` [1]. If the requirement that you're satisfying with this test is to ensure that the metadata is the same when the same repo is synced to multiple orgs, then I agree with Shweta that more than just the package count needs to be checked. Looking at the Repository Nailgun entity [2], it does not look like we're capturing all of that metadata in that entity. However, I would at least add checks against errata and packages to this test. @shweta83 were there other parts of the repodata that you were thinking of here? 

[1] https://docs.pulpproject.org/en/2.19/plugins/pulp_rpm/tech-reference/rpm.html
[2] https://github.com/SatelliteQE/nailgun/blob/84c8c95cedb2f319a710bee93f084ddf76d4bb84/nailgun/entities.py#L6407

Reply:
content_counts in this case returns a dictionary of rpm, erratum, package_groups, srpm, module_stream, etc. So we are asserting all of those are consistent between repos